src/status.js

```diff
@@ -0,0 +1,8 @@
+// eslint-disable-next-line import/no-cycle
+import { tasks } from './index.js';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
As part of the general requirements for this and all other projects, kindly do not disable the linters. The linters are there to guide you in order to produce the best standard practices. Kindly consider removing the lines 1 and 4 and work within the limits of the linters. :pray: